pkg/didcomm/protocol/presentproof/states.go

```diff
@@ -31,7 +32,10 @@ const (
 	stateNameProposalSent     = "proposal-sent"
 )
 
-const codeInternalError = "internal"
+const (
```

Comment:
any reason for declaring const twice?

Reply:
what do you mean?

Reply:
I think he's suggesting that you can combine the two const (...) blocks into one, unless there's a specific reason you separated the two

Reply:
Oh, I got it. Actually, there is no reason. Code just looks more readable.